pom.xml

```diff
@@ -173,13 +173,50 @@
       <scope>runtime</scope>
     </dependency>
 
+      <dependency>
```

Comment:
I don’t think we’re using JAXB, Mockito nor SLF4J for anything? I suggest not adding unused dependencies.